lib/errors.js

```diff
@@ -1,39 +1,73 @@
+'use strict';
```

Comment:
I'd also like the note that I tested this code very aggressively against the unit tests, and you can see yourself from the wercker run that all of this code *does* have strict mode enabled almost everywhere and passes all tests fine.